cli/commands/project_sync.go

```diff
@@ -71,18 +72,28 @@ func (cmd *ProjectSync) Commands() []cli.Command {
 	return []cli.Command{start, stop}
 }
 
+// Run before all commands to setup core services
+func (cmd *ProjectSync) Before(c *cli.Context) error {
+	if err := cmd.BaseCommand.Before(c); err != nil {
+		return err
+	}
+	cmd.Config = NewProjectConfig()
```

Comment:
Why not just do this at the start of `RunStart()` and `RunStop()`, we can make an `InitProjectConfig()`, then we dont have to deal with the BaseCommand shuffling.

Reply:
Sure, we can do an InitProjectConfig() approach.